src/main/java/com/adoptmeplus/enterprise/service/AdoptionServiceStub.java

```diff
@@ -6,21 +6,17 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import java.io.IOException;
-import com.adoptmeplus.enterprise.dto.Adoption;
-import com.adoptmeplus.enterprise.dto.Dog;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
 import java.util.List;
 
 @Service
 public class AdoptionServiceStub implements IAdoptionService{
 
+    private final IAdoptionDAO adoptionDAO;
```

Comment:
http://odrotbohm.de/2013/11/why-field-injection-is-evil/
https://www.baeldung.com/constructor-injection-in-spring

Field injection is not preferred.